### DISCUSSION TITLE
Found SteelSeries Arctis (1) Wireless! Failed to open requested device.  HID Error: (null)

### DISCUSSION THREAD

**Discussion by Slightom** [Sapd/HeadsetControl#200] [Q&A]
Hi, Great idea for program! :) I have arctis 1 wireless headphones. headsetcontrol -? showed me 3 available options:

Found SteelSeries Arctis (1) Wireless!
Supported capabilities:

* sidetone
* battery status
* inactive time

but when i tried headsetcontrol -s 128 or headsetcontrol -b i get:

Found SteelSeries Arctis (1) Wireless!
Failed to open requested device.
 HID Error: (null)

What can cause that problem?

**Comment by Sapd** [comment]
Hello,

can you try it with sudo?

Unfortunately, the implementation hid_error gives no meaningful errors on Linux.
